var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -10,11 +10,12 @@ class Sz(CMakePackage):
     """Error-bounded Lossy Compressor for HPC Data"""
 
     homepage = "https://collab.cels.anl.gov/display/ESR/SZ"
-    url      = "https://github.com/disheng222/SZ/archive/v2.1.8.0.tar.gz"
-    git      = "https://github.com/disheng222/sz"
+    url      = "https://github.com/szcompressor/SZ/releases/download/v2.1.10/sz-2.1.10.tar.gz"
```

Comment:
You changed from an archive (automatically generated) to a download (manually generated). Are the checksums for historic versions all the same?